client-course-schedulizer/src/utilities/helpers/readCSV.ts

```diff
@@ -95,7 +95,7 @@ export const csvStringToSchedule = (csvString: string): Schedule => {
     if (fields) {
       fields.forEach((field) => {
         const value = String(object[field]);
-        field = field.replace(/\s/g, "");
+        field = field.replace(/\s/g, "").replace("ï»¿", "");
```

Comment:
haha what is that?

Reply:
So I was importing my `Course Section Enrollment.csv`, and I noticed that the `Department` field wasn't being exported, and when I logged the field names in `readCSV`, the first one was `ï»¿Department`. (Note that `math-schedule.csv` did work). Apparently, some files have that at the beginning, but you can't see it in an editor: https://stackoverflow.com/questions/3255993/how-do-i-remove-%C3%AF-from-the-beginning-of-a-file

Reply:
Wow, you learn something new each day. I didn't know that. Thanks!